tpl/ase_gauss_ts_search.tpl.py

```diff
@@ -39,11 +39,7 @@
 mol.set_calculator(calc)
 
 try:
-    db2 = connect('{working_dir}/geoms2.db')
-    db2.write(mol, name=label) 
```

Comment:
Are these changes related to the UQ? Was it part of a test? Please check and remove if not needed.

Reply:
yes this should be changed. I changed in the working code on keck, but not Rosalind where the UQ code is. Its been updated.